dawn/setup.py

```diff
@@ -48,119 +89,107 @@ class CMakeBuild(build_ext):
     def run(self):
         assert all(isinstance(ext, CMakeExtension) for ext in self.extensions)
 
-        # Check if a recent version of CMake is present
-        cmake_executable = os.getenv("CMAKE_EXECUTABLE", default="cmake")
-        cmake_args = os.getenv("CMAKE_ARGS", default="").split(" ") or []
-        try:
-            out = subprocess.check_output([cmake_executable, "--version"])
-        except OSError:
-            raise RuntimeError(
-                "CMake must be installed to build the following extensions: "
-                + ", ".join(e.name for e in self.extensions)
+        dawn_build_dir = os.getenv("DAWN_BUILD_DIR", default=None)
+        py_protos = [x + "_pb2.py" for x in ("IIR/IIR", "SIR/SIR", "SIR/statements", "SIR/enums")]
```

Comment:
is this future-proof? can we not find files in the `SIR` / `IIR` subfolders?

Reply:
Sure, I can find files there instead. I'll fix in another commit.